state-chain/pallets/cf-elections/src/electoral_system.rs

```diff
@@ -397,3 +398,278 @@ pub use access::{
 	ConsensusStatus, ElectionReadAccess, ElectionWriteAccess, ElectoralReadAccess,
 	ElectoralWriteAccess,
 };
+
+#[cfg(test)]
+pub mod mocks {
```

Comment:
Not sure I understand why we don't just the usual substrate pallet mock runtime?

Reply:
These mocks are used to test the electoral systems which do not require any runtime access.